main.go

```diff
@@ -483,10 +524,8 @@ func newNotifier(kapi client.KeysAPI, path string, socksProxy string, etcdPeers
 
 			for err == nil {
 				_, err = watcher.Next(context.Background())
-				select {
-				case w.ch <- struct{}{}:
-				default:
-				}
+				w.ch <- struct{}{}
```

Comment:
I now realise the behaviour has changed significantly.
Is there any documentation of what the "buffer size" really means?